src/reformat_libyuv.c

```diff
@@ -555,6 +498,31 @@ avifResult avifImageYUVToRGBLibYUV8bpc(const avifImage * image,
             }
             return AVIF_RESULT_OK;
         }
+    } else if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV400) {
```

Comment:
Since we are checking for specific values of `image->yuvFormat` now, we should add an `else` branch and return `AVIF_RESULT_INVALID_ARGUMENT` or `AVIF_RESULT_REFORMAT_FAILED`.

On the other hand, if we reach this function, we already know `image->yuvFormat` is one of the four YUV formats in AV1 (it's been validated by the `avifPrepareReformatState()` call in `avifImageRGBToYUV()`, so the original `if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV400) { ... } else { ...}` structure is correct and simpler. We don't need to validate `image->yuvFormat` redundantly here.

Reply:
I believe following enum order is simpler to maintain, so I just removed `if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV400)`.